manifest.json

```diff
@@ -2,7 +2,7 @@
   "vendor": "vtex",
   "name": "minicart",
   "title": "Mini Cart",
-  "version": "2.48.0",
+  "version": "2.49.0",
```

Comment:
There is no need to update the version in the `manifest`, this is part of the deploy process performed by a bot :)